UnitySetup/UnitySetup.psm1

```diff
@@ -2347,7 +2352,8 @@ function Update-PackageAuthConfig {
         [int]$PATLifetime,
         [string]$DefaultScope,
         [string]$ScopedURLRegEx,
-        [string]$UPMRegEx
+        [string]$UPMRegEx,
+        [string]$defaultSubscriptionId
```

Comment:
I think `AzureSubscription` is a better and more concise name.

Reply:
Sounds good - updated